test/utils/DeployOptimized.sol

```diff
@@ -0,0 +1,77 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity >=0.8.19 <0.9.0;
+
+import { StdCheats } from "forge-std/StdCheats.sol";
+
+import { ISablierV2Comptroller } from "../../src/interfaces/ISablierV2Comptroller.sol";
+import { ISablierV2LockupDynamic } from "../../src/interfaces/ISablierV2LockupDynamic.sol";
+import { ISablierV2LockupLinear } from "../../src/interfaces/ISablierV2LockupLinear.sol";
+import { ISablierV2NFTDescriptor } from "../../src/interfaces/ISablierV2NFTDescriptor.sol";
+
+abstract contract DeployOptimized is StdCheats {
+    /// @dev Deploys {SablierV2Comptroller} from a optimized source compiled with `--via-ir`.
+    function deployOptimizedComptroller(address initialAdmin) internal returns (ISablierV2Comptroller) {
+        return ISablierV2Comptroller(
+            deployCode("out-optimized/SablierV2Comptroller.sol/SablierV2Comptroller.json", abi.encode(initialAdmin))
+        );
+    }
+
+    /// @dev Deploys {SablierV2LockupDynamic} from a optimized source compiled with `--via-ir`.
+    function deployOptimizedDynamic(
```

Comment:
Can we rename this to `deployOptimizedLockupDynamic`, please? So that "dynamic" is not mistaken for having something to do with the compilation itself.